### ISSUE TITLE
"New contact" text in name field shall be selected when clicking on it

### ISSUE THREAD

**Issue by johkoenig** [nextcloud/contacts#697]
### Describe the bug
When creating a new contact, the name `New contact` is prefilled. However, when clicking onto that, the text `New contact` doesn't vanish, it must be removed manually by the user. For `Organisation` and `Title`, this is not the case. When I click on one of these and start typing, they vanish. Only the typed text remains.

**To Reproduce**
1. Click `+ New contact`
2. Try to add the name by clicking into the name field, which is prefilled with `New contact`
3. Type `John Doe`

**Expected behavior**
`New contact` vanishes, name field is filled with `John Doe`
Basically, name should behave the same way as title and organisation

**Actual behavior**
The name field is filled with `New conJohn Doetact`

### Server configuration
**Nextcloud version:** 14.0.3
**Contacts version:** 2.1.6

**Comment by skjnldsv** [comment]
Hello, thanks a lot for your report!
This is done on purpose because of the way we create the 'new contact' contact on the background.
If clicking the input select the whole text, would this seems to be a proper fix for you?

**Comment by johkoenig** [comment]
Yes, this would be acceptable.
My excepted behavior is "click" -> "write" and not "click" -> "select" -> "delete" -> "write", which would be fulfilled if the text is selected on click.
